### ISSUE TITLE
A.05.IR11.custodian.contact.point

### ISSUE THREAD

**Issue by jensscheerlinck** [inspire-eu-validation/ats-interoperable-sds#11]
_This issue has been extracted from the issue list on:https://ies-svn.jrc.ec.europa.eu/issues/2685_
# Comment

Notes:
'àThe XPath test also requiresà'
Instead of saying that, the test should say that the Responsible Party must have passed also the corresponding test in the ats-metadata.

**Comment by cportele** [comment]
These SDS-related conformance classes / ATSs have not been part of the review: https://ies-svn.jrc.ec.europa.eu/issues/2685

Proposed resolution:
- Keep the issue open and revisit as part of the future work on the SDS ATS.

**Comment by cportele** [comment]
Changed to rejected and wontfix as this reflects the status during this review. The issue may be revisited once the SDS ATS is ready for review.
